<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30px &quot;Fredoka&quot;;
font-weight: 800;
qproperty-alignment: AlignCenter;
color: #BA8F4A</string>
   </property>
   <property name="text">
    <string>LaptopShield+</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;fredoka&quot;;
font-size : 15px;
font-weight: bold;
text-align:center;
color: #008000
</string>
   </property>
   <property name="text">
    <string>CONFIRM DETECTED OBJECTS</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/raphe/OneDrive/Pictures/PROJECT/UG.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>290</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/raphe/Downloads/balme4-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb_confirm_items">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="tabKeyNavigation">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Device Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Captured Image</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="objects_confirm">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/raphe/OneDrive/Pictures/PROJECT/done1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="obj">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/raphe/OneDrive/Pictures/PROJECT/cancel1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="objects_accept">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: None;
font-family: Fredoka;
font-size: 10pt;
font-weight:bold;
border: 1px solid #000;
border-radius: 5px;
background-color:#008000;
color: #fff;</string>
   </property>
   <property name="text">
    <string>Accurate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="objects_reject">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: None;
font-family: Fredoka;
font-size: 10pt;
font-weight:bold;
border: 1px solid #000;
border-radius: 5px;
background-color: #ff0000;
color: #fff</string>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
  <zorder>objects_confirm</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>tb_confirm_items</zorder>
  <zorder>obj</zorder>
  <zorder>objects_accept</zorder>
  <zorder>objects_reject</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
